Lesson-4-Task-2/src/main/java/Lesson4/Shape.java

```diff
@@ -0,0 +1,5 @@
+package Lesson4;
+public abstract class Shape {
+    protected abstract double getArea();
```

Comment:
dlaczego protected?

Reply:
Chciałem ograniczyć zakres do dziedziczenia.

Reply:
dlaczego? ;>
Skoro klase masz public i nie ma konkretnych przesłanek, by było inaczej to niej metody też będą publiczne.